res/scripts/celmi/timetables/timetable_helper.lua

```diff
@@ -96,8 +96,9 @@ function timetableHelper.startVehicle(vehicle)
     if type(vehicle) == "string" then vehicle = tonumber(vehicle) end
     if not(type(vehicle) == "number") then print("Expected String or Number") return false end
 
-    api.cmd.sendCommand(api.cmd.make.setVehicleShouldDepart(vehicle))
+    --api.cmd.sendCommand(api.cmd.make.setVehicleShouldDepart(vehicle))
     api.cmd.sendCommand(api.cmd.make.setUserStopped(vehicle,false))
```

Comment:
I think this line might unexpectedly restart a train that has been manually stopped by the user.

Reply:
Yes, ideally this line shouldn't be necessary anymore.
However, for backwards compatibility, on an old save, the trains that have previously been stopped by the mod, would stay stopped until manually reactivated. 
Also, only trains that are stopped within a station are affected by this